scripts/analysis.py

```diff
@@ -999,43 +967,45 @@ def stacked_bar_chart(dictionary, timestep,
             label = str(key)
         # very first country does not have a 'bottom' argument
         if sum(dictionary[key]) == 0:
-            print(str(dictionary[key]) + 'has no values')
+            print(label + ' has no values')
         elif top_index is True:
-            plot = plt.bar(left=init_year + (timestep/12),
+            plot = plt.bar(left=init_year + (timestep / 12),
                            height=dictionary[key],
                            width=0.1,
-                           color=cm.viridis(1.*color_index/len(dictionary)),
-                           edgecolor='none',
-                           label=label)
+                           color=cm.viridis(
+                1. * color_index / len(dictionary)),
```

Comment:
- why are you multiplying by 1. ? 
- If you want it to be a float, cast it as a float instead. ```float(color_index)```
- it is best practice to have a digit on either side of the decimal. In this case, 1.0 is stylistically preferred over " 1. "